### PR TITLE
[SKIP CI] xtensa-build-zephyr.py: important -v verbosity fixes

### PR DESCRIPTION
**Pull request by marc-hb**
The most important fix is the addition of the "flush" argument which
means are commands are now printed when they are run and not all at the
end when everything is done.

Commands run by this script are very few so they are always shown.  This
makes it possible to debug just this script without being completely
drown in build noise. All '-v's are passed to west.

Pass -v to `west sign`, not just to `west build`.

Drop the manual "-DCMAKE_VERBOSE_MAKEFILE=ON", west knows how to do that.

Print directories first so commands are aligned, more visible and can be
copied. Use shlex to quote whitespace.

Signed-off-by: Marc Herbert <marc.herbert@intel.com>

### CODE REVIEW

**Comment by marc-hb** [comment]
The isolated ICL error in https://sof-ci.01.org/sof-pr-viewer/#/build/PR5404/build8649158 is totally unrelated to this build script fixes. Another test passed in the same configuration

@kkarask any idea?

````
03:26:57.947     Exception TimeoutError occurred at 'C:\\work\\tools\\pyexecnetcache\\cavs_scripts-py\\utilities\\fw_loader_pci.py':224
                 FW not ready after 5.0 seconds!
```

**Comment by wszypelt** [comment]
@marc-hb 
I guess it might be the machine's fault, I will restart the settings and run these tests again

**Review by lgirdwood — APPROVED** [review]

**Comment by lgirdwood** [comment]
@aborisovich any comments ?

File: scripts/xtensa-build-zephyr.py [hunk 1]
```diff
@@ -192,10 +192,10 @@ def execute_command(command_args, stdin=None, input=None, stdout=None, stderr=No
 	errors=None, text=None, env=None, universal_newlines=None):
 	"""[summary] Provides wrapper for subprocess.run (matches its signature) that prints
 	command executed when 'more verbose' verbosity level is set."""
-	if args.verbose==2:
-		print_args = " ".join(command_args) if isinstance(command_args, list) else command_args
-		print_cwd = f"in: {cwd}" if cwd else f"in: {os.getcwd()}"
-		print(f"Running command: {print_args} {print_cwd}")
+	if args.verbose >= 0:
```
Comment (aborisovich) [inline scripts/xtensa-build-zephyr.py:195]: I really would hestitate printing all this without verbose mode. If you rebuild something X times and know how this works then it just spams the console for me. I would leave it with at least:
`if args.verbose >= 1:`
But I won't insist if you care, its probably more of a personal preference.

**Review by aborisovich — COMMENTED** [review]
Overall everything is great, I would just think over that directories and commands printing when no verbosity level specified at all. If you really wish to keep it this way just confirm and I will approve.

**Comment by marc-hb** [comment]
In theory I agree with you. In practice this adds 9 lines out of 260 (3%) when building _incrementally_ a _single platform_ at the verbosity level _zero_!! So this is really a drop in the ocean of SOF build noise.

I think we can easily fine tune this later if/when the global SOF noise gets addressed. For instance we could have a -1 "quiet mode", or print only the important commands.

**Comment by aborisovich** [comment]
> In theory I agree with you. In practice this adds 9 lines out of 260 (3%) when building _incrementally_ a _single platform_ at the verbosity level _zero_!! So this is really a drop in the ocean of SOF build noise.
> 
> I think we can easily fine tune this later; for instance if/when the global SOF noise gets addressed. For instance we could have a -1 "quiet mode", or print only the important commands.

Fine by me.

**Review by aborisovich — APPROVED** [review]

**Pull request merged**